airflow/kubernetes/kube_client.py

```diff
@@ -41,6 +44,13 @@ def _load_kube_config(in_cluster, cluster_context, config_file):
 def get_kube_client(in_cluster=conf.getboolean('kubernetes', 'in_cluster'),
                     cluster_context=None,
                     config_file=None):
+    """
+    Retrieves Kubernetes client
```

Comment:
```suggestion
    Retrieves Kubernetes client
    
```
I think there always should be one empty line between title and params

Reply:
and could you add `:type`s below, please :)